packages/addons-website/scss/components/_aspect-ratio-tile.scss

```diff
@@ -0,0 +1,129 @@
+@import '../prefix';
+
+.#{$prefix}--aspect-ratio-tile {
+  position: relative;
+  display: block;
+  width: 100%;
+  padding-bottom: 100%;
+  color: $carbon--gray-90!important;
+
+  .#{$prefix}--aspect-ratio-tile-background {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 1px;
+    bottom: 1px;
+    background-color: $carbon--gray-30;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    // z-index: 1;
+  }
+
+  .#{$prefix}--aspect-ratio-tile-content {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 1px;
+    bottom: 1px;
+    z-index: 3;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 1rem;
+    user-select: none;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    transition: opacity 600ms ease;
+
+    .#{$prefix}--aspect-ratio-tile-title {
+      z-index:2;
+    }
+    .#{$prefix}--aspect-ratio-tile-identity-icon {
+      position: absolute;
+      left: 1rem;
+      bottom: 0;
+      width: 50px;
+      height: 50px;    
+    }
+  }
+
+  .#{$prefix}--aspect-ratio-tile-hover {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 1px;
+    bottom: 1px;
+    z-index: 2;
+    background: rgba(255,255,255,.08);
+    opacity: 0;
+    transition: opacity 600ms ease;
```

Comment:
We have tokens for animation, I think 

`transition: opacity $transition--base $carbon--standard-easing;`

or just `transition: opacity $transition--base` might work